opal-r/src/main/java/org/obiba/opal/web/r/Dtos.java

```diff
@@ -54,4 +56,15 @@ private Dtos() {}
         .build();
   }
 
+
+  public static Ws.ClientErrorDto getErrorMessage(Response.StatusType status,
+                                                  String errorStatus,
+                                                  RuntimeException exception) {
+    return Ws.ClientErrorDto.newBuilder()
+            .setStatus(errorStatus)
+            .setCode(status.getStatusCode())
+            .addArguments(exception.getMessage())
```

Comment:
Message is not cleaned as it was before.

Reply:
_ is not cleaned _ : elaborate please.

Reply:
Line 28 you removed in REvaluationRuntimeExceptionMapper.java